demo/arxiv_dataset/base.py

```diff
@@ -0,0 +1,51 @@
+import os
+import json
+from pathlib import Path
+
+from sqlalchemy import text
+
+
+def setup_and_register_schema_for_receiving_arxiv_data(engine):
+    db_name, schema_name = _setup_arxiv_schema(engine)
+    _make_sure_parent_directories_present(get_arxiv_db_and_schema_log_path())
+    _append_db_and_schema_to_log(db_name, schema_name)
+
+
+def _make_sure_parent_directories_present(path_to_file):
+    path_to_file = Path(path_to_file)
+    parent_directory_of_file = path_to_file.parent
+    parent_directory_of_file.mkdir(parents=True, exist_ok=True)
+
+
+def _append_db_and_schema_to_log(db_name, schema_name):
+    path = get_arxiv_db_and_schema_log_path()
+    db_and_schema = [db_name, schema_name]
+    with open(path, 'a') as f:
+        json.dump(db_and_schema, f)
+        f.write('\n')
+
+
+def get_arxiv_db_and_schema_log_path():
+    return os.path.abspath(
+        '/var/lib/mathesar/demo/arxiv_db_schema_log'
+    )
+
+
+def _setup_arxiv_schema(engine):
+    schema_name = 'Arxiv'
```

Comment:
This needs to be a parameter passed from `demo.install`. The reason is that we have other things in the work that reset and manage these demo schemata, and we need their names consolidated there.

Reply:
Done here https://github.com/centerofci/mathesar/pull/2212/commits/c55529dca3d39c040d29bc659c6a8f286f5db3c1.